storage/s3.go

```diff
@@ -737,11 +737,18 @@ func newCustomRetryer(maxRetries int) *customRetryer {
 // ShouldRetry overrides SDK's built in DefaultRetryer, adding custom retry
 // logics that are not included in the SDK.
 func (c *customRetryer) ShouldRetry(req *request.Request) bool {
-	if errHasCode(req.Error, "InternalError") || errHasCode(req.Error, "RequestTimeTooSkewed") {
-		return true
+	shouldRetry := errHasCode(req.Error, "InternalError") || errHasCode(req.Error, "RequestTimeTooSkewed")
+	if !shouldRetry {
+		shouldRetry = c.DefaultRetryer.ShouldRetry(req)
 	}
 
-	return c.DefaultRetryer.ShouldRetry(req)
+	if shouldRetry && req.Error != nil {
+		err := fmt.Errorf("retryable error: %v", req.Error)
+		msg := log.ErrorMessage{Err: err.Error()}
```

Comment:
We should definitely log the message but I'm not sure about the severity since we still retry.

Reply:
Maybe Debug? We don't actively use this severity level 

only here: https://github.com/peak/s5cmd/blob/358a6c4261ec19d4f7305da9129dd0d6c65441c9/command/cp.go#L423

Reply:
debug is 👍

Reply:
done